src/App.js

```diff
@@ -1,8 +1,27 @@
 import React from 'react';
 import './App.scss';
+import PropTypes from 'prop-types';
 
-const App = () => (
-  <h1>React sum</h1>
+const App = ({ a, b }) => (
+  <p>
+    Sum
+    of
+    {' '}
+    {a}
+    {' '}
+    and
+    {' '}
+    {b}
+    {' '}
+    is
+    {' '}
+    {a + b}
```

Comment:
There is much easier way to render this text:
```suggestion
    {`Sum of ${a} and ${b} is ${a + b}`}
```